.github/workflows/pylint_check.yaml

```diff
@@ -4,10 +4,12 @@ on:
     branches:
       - master
       - develop
+      - remove_dependencies
```

Comment:
```suggestion
```
You shouldn't need to do this. Running pylint on your system with the pylint.rc file should have your code pass the PR linting.

Reply:
The above comment does pre-suppose that there're no changes necessary to the .yaml workflow. It also assumes that the requirements.txt and packages.txt satisfies the pylint requirements